### PR TITLE
Add datasets

### PR DESCRIPTION
**Pull request by slesaad**
# Changes
- Added dataset definitions for datasets needed for EIS discoveries

### CODE REVIEW

File: data/datasets/lis-da-gpp-trend.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    ],
+    "discovery_items": [
+      {
+        "discovery": "s3",
+        "cogify": false,
+        "upload": false,
+        "dry_run": false,
+        "prefix": "EIS/COG/LIS_GLOBAL_DA/DA_Trends/",
+        "bucket": "veda-data-store-staging",
+        "filename_regex": "(.*)DAGPP_STL_based_trendv2_2003_2021.cog.tif$",
```
Comment (moradology) [inline data/datasets/lis-da-gpp-trend.json:29]: Should change to "(.*)DAGPP_STL_based_trend_v3_2003_2021.cog.tif$"

File: data/datasets/lis-da-tws-trend.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  ],
+  "discovery_items": [
+    {
+      "discovery": "s3",
+      "cogify": false,
+      "upload": false,
+      "dry_run": false,
+      "prefix": "EIS/COG/LIS_GLOBAL_DA/DA_Trends/",
+      "bucket": "veda-data-store-staging",
+      "filename_regex": "(.*)DATWS_STL_based_trendv2_2003_2021.cog.tif$",
```
Comment (moradology) [inline data/datasets/lis-da-tws-trend.json:29]: Should change to "(.*)DATWS_STL_based_trend_v3_2003_2021.cog.tif$"

**Review by moradology — CHANGES_REQUESTED** [review]
A couple small edits to point to updated files

**Comment by gadomski** [comment]
This repo is being sunsetted (https://github.com/NASA-IMPACT/veda-data-pipelines/pull/360), so we are closing all open PRs. Please re-open on https://github.com/NASA-IMPACT/veda-data if needed.

**Pull request closed without merging**